### DISCUSSION TITLE
Update from 14.1.4 to 14.2.1 several .css files instead of one

### DISCUSSION THREAD

**Discussion by welschmoor** [vercel/next.js#64498] [App Router]
What is the reason behind it? "All" my life I saw a single .css file when using Next.js, and after update I see three. 

<img width="979" alt="Screenshot 2024-04-15 at 12 48 20" src="https://github.com/vercel/next.js/assets/87126711/cacaf4cd-211f-4feb-8e14-1b63e472c35d">

**Comment by icyJoseph** [comment]
I think they, somewhat, expand on this here: https://nextjs.org/blog/next-14-2#css

**Reply by welschmoor** [reply]
Saw three .css files under render blocking resources in a lighthouse test; it did scare me a bit at first. No way to opt out it seems. And now I have to cache 3 files in cdn instead of one?
